vulnerability/worker.go

```diff
@@ -79,6 +79,8 @@ func NewVulnerabilityWorker(log *logp.Logger, c *config.Config) (*VulnerabilityW
 func (f *VulnerabilityWorker) Run(ctx context.Context) {
 	// TODO: Handle deletion of snapshots
 	f.log.Info("Starting VulnerabilityWorker.work")
+	// TODO: Maybe use the context from the worker? But what if the worker is canceled?
```

Comment:
What would happen if the worker is canceled and this continues to run?
imh it should be context from worker